<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wherefore</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="consumerBarLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="brokerAddressLayout">
          <item>
           <widget class="QLabel" name="brokerAddressLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Broker address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="brokerAddressEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line1_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="startAtLabel">
          <property name="text">
           <string>Start at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="startAtSelector"/>
        </item>
        <item>
         <widget class="QLineEdit" name="startOffsetEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="startTimeEdit"/>
        </item>
        <item>
         <widget class="Line" name="line1_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="endAtLabel">
          <property name="text">
           <string>Stop at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="endAtSelector"/>
        </item>
        <item>
         <widget class="QLineEdit" name="endOffsetEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="endTimeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="kafkaConfigFileLayout">
        <item>
         <widget class="QLabel" name="kafkaConfigFileLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Configuration file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="kafkaConfigFileValueLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>(unset)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="topicPartitionSourceTree"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="enableDisableLayout">
          <item>
           <widget class="QPushButton" name="enableAllButton">
            <property name="text">
             <string>Enable all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disableAllButton">
            <property name="text">
             <string>Disable all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="enableDefaultComboBox">
            <property name="placeholderText" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="partitionInfoBox">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Partition info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="partitionInfoLayout">
             <property name="spacing">
              <number>-1</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QFormLayout" name="lowLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="lowOffsetLabel">
                 <property name="text">
                  <string>Low offset:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="lowOffsetValue">
                 <property name="text">
                  <string>n/a</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line3_1">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="highLayout">
               <item row="1" column="0">
                <widget class="QLabel" name="highOffsetLabel">
                 <property name="text">
                  <string>High offset:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="highOffsetValue">
                 <property name="text">
                  <string>n/a</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line3_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="lagLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="lagLabel">
                 <property name="text">
                  <string>Current lag:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="lagValue">
                 <property name="text">
                  <string>n/a</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="sourceInfoBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Source info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,0,0">
           <property name="spacing">
            <number>-1</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0,0">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <property name="horizontalSpacing">
                <number>-1</number>
               </property>
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="sourceNameLabel">
                   <property name="text">
                    <string>Source name:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="sourceNameValue">
                   <property name="text">
                    <string>n/a</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="sourceTypeLabel">
                   <property name="text">
                    <string>Source type:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="sourceTypeValue">
                   <property name="text">
                    <string>n/a</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="otherLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="firstOffsetLabel">
               <property name="text">
                <string>First offset</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="firstOffsetValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="currentOffsetLabel">
               <property name="text">
                <string>Current offset</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="currentOffsetValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="currentMsgSizeLabel">
               <property name="text">
                <string>Current msg. size</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="currentMsgSizeValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="receivedMessagesLabel">
               <property name="text">
                <string>Received messages</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="receivedMessagesValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="consumptionRateLabel">
               <property name="text">
                <string>Consumption rate</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="consumptionRateValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="messageRateLabel">
               <property name="text">
                <string>Message rate</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="messageRateValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="dataRateLabel">
               <property name="text">
                <string>Data rate</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="dataRateValue">
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="dataLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Data:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="dataValue">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>n/a</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="timestampsBox">
             <property name="title">
              <string>Timestamps</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="firstMsgTimLabel">
                <property name="text">
                 <string>First message time</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="firstMsgTimeValue">
                <property name="text">
                 <string>n/a</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lastMsgReceiveTimeLabel">
                <property name="text">
                 <string>Last received time</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="lastMsgReceiveTimeValue">
                <property name="text">
                 <string>n/a</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lastMsgTimeLabel">
                <property name="text">
                 <string>Last message time</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="lastMsgTimeValue">
                <property name="text">
                 <string>n/a</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lastMsgKafkaTimeLabel">
                <property name="text">
                 <string>Last Kafka time</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="lastMsgKafkaTimeValue">
                <property name="text">
                 <string>n/a</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
